Twitter login doc update (#3221)
wso2/docs-is: en/docs/guides/identity-federation/twitter.md

> 1. Login to <https://developer.twitter.com/> using your Twitter credentials. 

Reviewer: ```suggestion
1. Log in to <https://developer.twitter.com/> using your Twitter credentials. 
```

> 2. Click on **Create  project** and set up your project.

Reviewer: ```suggestion
2. Click **Create  project** and set up your project.
```

    
    ![Create Twittter project]({{base_path}}/assets/img/samples/twitter-dashboard.png)

> 3. In the **App Set Up**.

Reviewer: ```suggestion
3. In **App Set Up**, follow the steps given below.
```

{!./includes/register-a-service-provider.md !}

4. In the **Inbound Authentication Configuration** section, click
   **Configure** under the **OAuth/OpenIDConnect Configuration** section and
   set the configurations as required.

> 5. Configure the Callback URL of the sample application (**http://localhost.com:8080/pickup-dispatch/oauth2client**).

Reviewer: ```suggestion
5. Configure the Callback URL of the sample application (`http://localhost.com:8080/pickup-dispatch/oauth2client`).
```

   ![configuring-OAuth-Callback-URL]({{base_path}}/assets/img/guides/oauth-callback-url.png)

> 6. Keep the other configurations as default and click on **Add**

Reviewer: ```suggestion
6. Keep the other configurations as default and click **Add**
```

7. Click **Register**. Now you will be sent back to the **Service Providers** page.
8. Take a copy of the **OAuth Client Key** and the **OAuth Client Secret** for later usages

   ![OAuth-id-secret]({{base_path}}/assets/img/guides/oauth-id-secret.png)

9. Go to the **Local and Outbound Authentication Configuration**
   section.

> 10. For **Authentication Type**, select the **Federated Authentication** radio button and select the

Reviewer: ```suggestion
10. For **Authentication Type**, select **Federated Authentication**  and select the
```
